mods/persistence/modules/organs/internal/stack.dm

```diff
@@ -19,9 +19,10 @@ var/global/list/cortical_stacks = list()
 	parent_organ = BP_HEAD
 	icon_state = "cortical-stack"
 	organ_tag = BP_STACK
-	vital = 1
+	vital = TRUE
```

Comment:
Note for later discussion: Cortical stacks probably shouldn't be vital. Ripping them out of your head shouldn't do anything except get rid of cortical chat, since you don't "shunt" into them until you die

Reply:
I thought about it. Problem being that, if someone slaps in a stack, removes it, and then repeats with a bunch of stacks, you end up essentially being immortal as long as you got a stack with your backup left on it? Not sure if that's intentional or not?